docs/src/main/sphinx/admin/properties-client-protocol.md

```diff
@@ -0,0 +1,183 @@
+# Client protocol properties
+
+The following sections provide a reference for all properties related to the
+[client protocol](/client/client-protocol). 
+
+(prop-protocol-spooling)=
+## Spooling protocol
+
+The following properties are related to the [](protocol-spooling).
+
+### `protocol.spooling.enabled`
+
+- **Type:** [](prop-type-boolean)
+- **Default value:** `true` (is that automatically configured if shares secret or so is set up)
+
+Enable spooled client protocol support. 
+
+### `protocol.spooling.shared-secret-key`
+
+- **Type:** [](prop-type-string)
+
+256 bit, base64-encoded secret key used to secure segment identifiers.
```

Comment:
I'd scratch "segment identifiers" and just wrote "to secure spooled metadata exchanged with the client"

Reply:
used this but not metadata .. since I think all data is encrypted with this key.. right? Not just the metadata.

`to secure spooled data exchanged with the client`

Reply:
no, data is encrypted with ephemeral keys, this key is not used for that

Reply:
this is to secure the segment identifier that is part of the URI, the identifier is a stateless representation of the object+ttl and other information

Reply:
so metadata, not data

Reply:
ok .. thanks for clarifying